CI fail:  verify:  golangci-lint not found

<!-- 
STOP -- PLEASE READ!

If you're looking for help, check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
Have questions? First please read [Frequently Asked Questions](https://github.com/kubernetes-sigs/metrics-server/blob/master/FAQ.md)
Encountered a problem? First please read [Known Issues](https://github.com/kubernetes-sigs/metrics-server/blob/master/KNOWN_ISSUES.md)
You can also post your question on the [#sig-instrumentation](https://kubernetes.slack.com/messages/sig-instrumentation) channel of [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.
If the matter is security related, please disclose it privately via https://kubernetes.io/security/.

Please use template below and provide as much info as possible.
Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->

**What happened**:
Run `pull-metrics-server-verify`, an error occurred
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_metrics-server/1033/pull-metrics-server-verify/1531881722621202432
```
curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b /home/prow/go/bin latest
golangci-lint run --timeout [10](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_metrics-server/1033/pull-metrics-server-verify/1531881722621202432#1:build-log.txt%3A10)m --modules-download-mode=readonly || (echo 'Run "make update"' && exit 1)
/bin/sh: 1: golangci-lint: not found
Run "make update"
make: *** [Makefile:216: verify-lint] Error 1
```
It should be that `https://install.goreleaser.com/github.com/golangci/golangci-lint.sh` cannot be accessed.
The following URL can be accessed
`https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh`

**What you expected to happen**:
 `pull-metrics-server-verify` success
**Anything else we need to know?**:

**Environment**:
- Kubernetes distribution (GKE, EKS, Kubeadm, the hard way, etc.):
- Container Network Setup (flannel, calico, etc.):
- Kubernetes version (use `kubectl version`):

- Metrics Server manifest

<details>
  <summary>spoiler for Metrics Server manifest:</summary>

  <!--- INSERT manifest HERE --->

</details>

- Kubelet config:

<details>
  <summary>spoiler for Kubelet config:</summary>

  <!--- INSERT kubelet config HERE --->

</details>

- Metrics server logs:

<details>
  <summary>spoiler for Metrics Server logs:</summary>

  <!--- INSERT logs HERE --->

</details>

- Status of Metrics API:
<details>
  <summary>spolier for Status of Metrics API:</summary>

  ```
  kubectl describe apiservice v1beta1.metrics.k8s.io
  ```

  <!--- INSERT results of command above --->

</details>

<!-- DO NOT EDIT BELOW THIS LINE -->
/kind bug

--- a/Makefile
+++ b/Makefile
@@ -222,7 +222,7 @@ HAS_GOLANGCI:=$(shell which golangci-lint)
 .PHONY: golangci
 golangci:
 ifndef HAS_GOLANGCI
-	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin latest
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin latest
 endif
 
 # Table of Contents

--- a/Makefile
+++ b/Makefile
@@ -222,7 +222,7 @@ HAS_GOLANGCI:=$(shell which golangci-lint)
 .PHONY: golangci
 golangci:
 ifndef HAS_GOLANGCI
-	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin latest
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.46.2
 endif
 
 # Table of Contents

